pkg/networkservice/ipam/ip_pool.go

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ipam
+
+import (
+	"encoding/binary"
+	"net"
+	"sync"
+
+	"github.com/RoaringBitmap/roaring"
+	"github.com/pkg/errors"
+
+	"github.com/networkservicemesh/sdk/pkg/tools/cidr"
+)
+
+var (
+	p2pMask net.IPMask = []byte{0xFF, 0xFF, 0xFF, 0xFE}
```

Comment:
Can we make this a private function to prevent static data invalidation?

Reply:
Yep, fixed.